languages/ruby/lib/oso/polar/query.rb

```diff
@@ -104,12 +104,37 @@ def handle_call(attribute, call_id:, instance:, args:)
         call_result(nil, call_id: call_id)
       end
 
+      def handle_make_external(data) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
+        id = data['instance_id']
+        raise DuplicateInstanceRegistrationError, id if host.instance? id
+
+        constructor = data['constructor']['value']
+        if constructor.key? 'InstanceLiteral'
```

Comment:
It looks like this PR removes the ability to parse this, no? Or is this still used for patterns or something?

Reply:
Ah yeah I forgot about the parser.